README.md

```diff
@@ -76,1299 +76,1297 @@ Additionally, you can check out [#37](https://github.com/pirate/sites-using-clou
 
 ## Notable Sites
 
-- authy.com
-- coinbase.com
-- bitcoin.de
-- betterment.com
-- transferwise.com
-- prosper.com
-- digitalocean.com ([no leaked data found in several search engine caches](https://www.digitalocean.com/company/blog/digitalocean-your-data-and-the-cloudflare-vulnerability/))
-- patreon.com
-- bitpay.com
-- news.ycombinator.com
-- producthunt.com
-- medium.com
-- 4chan.org
-- yelp.com
-- okcupid.com
-- zendesk.com
-- uber.com
-- poloniex.com
-- localbitcoins.com
-- kraken.com
-- 23andme.com
-- curse.com (and some other Curse sites like minecraftforum.net)
-- counsyl.com
-- tfl.gov.uk
-- account.leagueoflegends.com
-- myaccount.nytimes.com
-- technicpack.net
-- cloudflare.com
-- namecheap.com ([no leaked data found in several search engine caches](https://status.namecheap.com/archives/30660))
-- discordapp.com ([affected](https://blog.discordapp.com/safety-jim-psa-cloudflare-security-issue-77a4ecc48298))
-- glassdoor.com ([no leaked data found in several search engine caches](https://twitter.com/Glassdoor/status/835238343822589952))
-- vultr.com ([no leaked data found in several search engine caches](https://github.com/pirate/sites-using-cloudflare/issues/130))
-- fastmail.com ([not affected](https://twitter.com/FastMail/status/834939787924557824), [#2](https://news.ycombinator.com/item?id=13720050))
-- 1password.com ([not affected](https://discussions.agilebits.com/discussion/comment/356869/#Comment_356869))
+- [authy.com](http://authy.com)
+- [coinbase.com](http://coinbase.com)
+- [bitcoin.de](http://bitcoin.de)
+- [betterment.com](http://betterment.com)
+- [transferwise.com](http://transferwise.com)
+- [prosper.com](http://prosper.com)
+- [digitalocean.com](https://digitalocean.com) ([no sensitive data found in multiple search engine caches](https://www.digitalocean.com/company/blog/digitalocean-your-data-and-the-cloudflare-vulnerability/))
```

Comment:
Why was this changed to "no sensitive data found in multiple search engine cashes" instead of "no leaked data found in several search engine caches" Just curious?